packages/nx/src/command-line/graph/graph.ts

```diff
@@ -246,6 +246,12 @@ export async function generateGraph(
   graph = filterGraph(graph, args.focus || null, args.exclude || []);
 
   if (args.file) {
```

Comment:
We need to alter this output to include `tasks` and omit `affectedProjects` and `criticalPath`.

Option 1:
`tasks` is the same as what `print-affected` has currently which is `tasks: Task[]`

Option 2:
`tasks` is the task graph so we have `taskGraph: { roots: string[], tasks: Record<string, Task>, dependencies: Record<string, string[]> }`

Reply:
- Updated with option 2 🎉 

There is a caveat. `print-affected` included task hashes, but they were sometimes not matching with the task hash as compared against the hash when the task included a custom hasher. The new implementation does not contain the hash at all.

Should we try to get accurate hashes this time, or just be ok with no hash?